src/App.js

```diff
@@ -23,21 +24,37 @@ const dummyMovie =  {id: 1,
 
 function App() {
 
-  const [movies, setMovies] = useState(movieData.movies)
+  const [movies, setMovies] = useState([])
   const [singleMovie, setSingleMovie] = useState(null)
 
-  function getMovieDetail() {
-    // i think the fetch will live here - fetch('url {id}')
-      setSingleMovie(dummyMovie)
-      // console.log(movies, "movies")
+  function getAllMovieData(){
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(response => response.json())
+    .then(data => setMovies(data.movies))
+    .catch(error => alert(`${error} please try again later`))
```

Comment:
I recommend using an 'error' state here so that you can manage and display errors to your users more effectively. Using an alert box isn't very user-friendly; it's better to render an error component or display an error message directly when an error occurs.